README.md

```diff
@@ -117,12 +117,10 @@ npm test
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](http://ridaajroud.me/math-magicians/issues).
+Feel free to check the [issues page](http://ridaajroud.me/Bookstore/issues).
 
```

Comment:
- [OPTIONAL ] Your README is informative, but I noticed repetition in the contribution section. Consider condensing this info to keep the README concise and engaging.
![Screenshot from 2023-08-31 20-23-04](https://github.com/AjroudiRida/bookstore/assets/88894525/dbb0318b-138e-4336-95c1-1881347390fd)